src/routes/volume/VolumeBulkActions.js

```diff
@@ -6,14 +6,43 @@ import style from './VolumeBulkActions.less'
 const confirm = Modal.confirm
 
 function bulkActions({ selectedRows, engineImages, bulkDeleteVolume, showBulkEngineUpgrade, showBulkChangeVolume, showBulkAttachHost, bulkDetach, bulkBackup, bulkExpandVolume, createPVAndPVC, createSchedule, confirmDetachWithWorkload, commandKeyDown }) {
+  const deleteWranElement = (rows) => {
+    let workloadResources = []
+    let pvResources = []
+
+    rows.forEach((item) => {
+      if (item && item.kubernetesStatus && item.kubernetesStatus.pvStatus) {
+        pvResources.push((<div style={{ margin: '10px', color: '#b7b7b7' }}>
+          <div style={{ paddingLeft: '10px' }}>PV Name: {item.kubernetesStatus.pvName}</div>
+          <div style={{ paddingLeft: '10px' }}>PVC Name: {item.kubernetesStatus.pvcName}</div>
+        </div>))
+      }
+      if (item.kubernetesStatus && item.kubernetesStatus.workloadsStatus && item.kubernetesStatus.workloadsStatus.length > 0) {
+        workloadResources.push((<div>
+          <div style={{ margin: '10px 0px', color: '#b7b7b7' }}>{item.kubernetesStatus.workloadsStatus.map((ele, i) => {
+            return (<div key={i} style={{ paddingLeft: '10px' }}>Pod Name: {ele.podName} ({ele.podStatus})</div>)
+          })}</div>
+        </div>))
+      }
+    })
+    return (<div>
+      <div>Any volumes currently attached to a node may cause errors in any applications using them once they are deleted!</div>
+      {workloadResources.map((item, i) => <div key={i}>{item}</div>)}
+      <div>Any resources (PersistentVolumes and PersistentVolumeClaims) associated with these volumes will be deleted.</div>
```

Comment:
Same deal here. If you can conditionally display the message (so it will only be displayed if there are resources that will be deleted), change it to:
```
The following resources (PersistentVolumes and PersistentVolumeClaims) associated with these volumes will be deleted:
```
If not, the current message is fine.